packages/fetch-node-details/test/nodeDetail.test.ts

```diff
@@ -20,6 +20,7 @@ describe("Fetch Node Details", function () {
     deepStrictEqual(details, getSapphireNodeDetails(TORUS_SAPPHIRE_NETWORK.SAPPHIRE_MAINNET, TORUS_LEGACY_NETWORK.MAINNET));
     strictEqual(metadataUrl, METADATA_MAP[TORUS_LEGACY_NETWORK.MAINNET]);
   });
+
```

Comment:
Should we add a test that for different key types different tss server endpoints are returned?